charts/prometheus-adapter/templates/configmap.yaml

```diff
@@ -4,11 +4,13 @@ kind: ConfigMap
 metadata:
   name: {{ template "k8s-prometheus-adapter.fullname" . }}
   namespace: {{ .Release.Namespace }}
+  {{- if .Values.customAnnotations }}
+  annotations:
+  {{- toYaml .Values.customAnnotations | nindent 4 }}
+  {{- end }}
   labels:
     app: {{ template "k8s-prometheus-adapter.name" . }}
```

Comment:
this is redundant with: "app.kubernetes.io/part-of"

Reply:
done